torch/fx/symbolic_trace.py

```diff
@@ -88,7 +88,9 @@ def create_arg(self, a: Any) -> 'Argument':
                     return self.create_node('get_attr', n, (), {})
             raise NameError('parameter is not a member of this module')
         elif isinstance(a, torch.Tensor):
-            for n, p in self.root.named_buffers():
+            # Type ignore here is required because "n" and "p" have different types
+            # a few lines above
+            for n, p in self.root.named_buffers():  # type: ignore[assignment]
```

Comment:
why cant we just use different variable names?

Reply:
Yes, that works too